docs/Wallet.html

```
@@ -1335,6 +1335,231 @@ <h5>Parameters:</h5>
+
+
+
+            <tr>
+
+              <td class="name"><code>labels</code></td>
+
+
+              <td class="type">
+
+
+                <span class="param-type">Array.&lt;String></span>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)